cli/src/main.rs

```diff
@@ -62,6 +62,14 @@ fn run() -> Result<(), CliError> {
                 (@arg quiet: -q --quiet "do not display output")
             )
         )
+        (@subcommand health =>
+            (about: "displays information about network health")
+            (@subcommand status =>
+                (about: "displays a node's version, endpoint, node id, and a list \
+                of the endpoints of its connected peers")
+                (@arg url: -u +takes_value "URL of node")
```

Comment:
Add the long argument '--url'. Not sure if this should be -U or -u. I seem to recall -U being more popular for this but haven't found specific examples yet.

Reply:
Should we say "URL of node's health REST API" to clarify which "URL"?